Makefile

```diff
@@ -357,6 +357,8 @@ notices:
 	./scripts/append_license.sh frontend/app/src/assets/fonts/Source_Code_Pro/Source-Code-Pro.LICENSE
 	./scripts/append_license.sh frontend/app/src/assets/fonts/Source_Sans_Pro/Source-Sans-Pro.LICENSE
 	./scripts/append_license.sh frontend/app/src/assets/fonts/Source_Serif_Pro/Source-Serif-Pro.LICENSE
+	./scripts/append_license.sh e2e_playwright/theming/static/Inter.LICENSE
+	./scripts/append_license.sh e2e_playwright/theming/static/NotoSans.LICENSE
```

Comment:
I think the NOTICES file only needs to contain licenses for assets that are distributed in our wheel. It's probably fine to just keep the two LICENSE files in our e2e static folder as an indication about the licenses.